nilearn/_utils/glm.py

```diff
@@ -1,30 +1,31 @@
-""" Misc utilities for the library
+"""Misc utilities for the library.
```

Comment:
Aren't these supposed to be utilities for the GLM module ?

Reply:
yup see #3661 
planning to tackle it after this PR and #3672 are in